src/Extension/Console/ExtensionDisableCommand.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension\Console;
+
+use Flarum\Console\AbstractCommand;
+use Flarum\Console\AskQuestionTrait;
+use Flarum\Extension\ExtensionManager;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class ExtensionDisableCommand extends AbstractCommand
+{
+    use AskQuestionTrait;
+
+    /**
+     * @var ExtensionManager
+     */
+    protected $extensions;
+
+    /**
+     * @param ExtensionManager $extensions
+     */
+    public function __construct(ExtensionManager $extensions)
+    {
+        $this->extensions = $extensions;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('extensions:disable')
+            ->setDescription('Disable an extension')
+            ->addArgument(
+                'extensions',
+                InputArgument::IS_ARRAY | InputArgument::OPTIONAL,
+                'The extensions to disable. You must provide this in extension-id format, that is: VENDOR_NAME-EXTENSION_NAME, based on the composer package, where "flarum-" or "flarum-ext-" is stripped from the extension name. For example, "flarum\tags" has an id of "flarum-tags", and "some-programmer/flarum-extension" has an id of "some-programmer-extension"'
```

Comment:
Do we need such an extensive description ? We should just document somewhere what a "Flarum extension ID" is (maybe we already do?) then assume the user is familiar with that definition.

Reply:
Yeah good point.